packages/lab/src/__tests__/__e2e__/form-field-next/FormFieldNext.cy.tsx

```diff
@@ -0,0 +1,178 @@
+          <MockChildren />
+        </FormFieldNext>
+      );
+      cy.findByText("Label").should("have.attr", "disabled");
+      cy.findByText("Helper text").should("have.attr", "disabled");
+    });
+  });
+
+  describe("WHEN readonly", () => {
+    it("THEN helper text should have readOnly set from useFormFieldNextProps.a11yProps", () => {
+      cy.mount(
+        <FormFieldNext readOnly>
+          <MockChildren />
+        </FormFieldNext>
+      );
+
+      cy.findByTestId("form-field-test").should("have.attr", "readonly");
+    });
+  });
+
+  describe("WHEN has error validationStatus", () => {
+    it("THEN StatusIndicator should show within Helper Text", () => {
+      cy.mount(
+        <FormFieldNext validationStatus="error">
+          <MockChildren />
+        </FormFieldNext>
+      );
+
+      cy.get(".saltStatusIndicator").should(
+        "have.class",
+        "saltStatusIndicator-error"
+      );
+    });
+
+    describe("AND is disabled", () => {
+      it("THEN the StatusIndicator should not show", () => {
+        cy.mount(
+          <FormFieldNext disabled validationStatus="error">
+            <MockChildren />
+          </FormFieldNext>
+        );
+
+        cy.get(".saltStatusIndicator").should(
+          "not.have.class",
+          "saltStatusIndicator-error"
+        );
+      });
+    });
+
+    describe("AND is readonly", () => {
+      it("THEN the StatusIndicator should not show", () => {
+        cy.mount(
+          <FormFieldNext readOnly validationStatus="error">
+            <MockChildren />
+          </FormFieldNext>
+        );
+
+        cy.get(".saltStatusIndicator").should(
+          "not.have.class",
+          "saltStatusIndicator-error"
+        );
+      });
+    });
+
+    describe("OR is success state", () => {
+      it("THEN the success variant should show", () => {
+        cy.mount(
+          <FormFieldNext readOnly validationStatus="success">
+            <MockChildren />
+          </FormFieldNext>
+        );
+
+        cy.get(".saltStatusIndicator").should(
+          "have.class",
+          "saltStatusIndicator-success"
+        );
+      });
+    });
+
+    describe("OR is warning state", () => {
+      it("THEN the warning variant should show", () => {
+        cy.mount(
+          <FormFieldNext readOnly validationStatus="warning">
+            <MockChildren />
+          </FormFieldNext>
+        );
+
+        cy.get(".saltStatusIndicator").should(
+          "have.class",
+          "saltStatusIndicator-warning"
+        );
+      });
+    });
+  });
+
+  describe("WITH a nested InputNext", () => {
+    it("SHOULD have no a11y violations on load", () => {
+      cy.mount(
+        <FormFieldNext>
+          <FormFieldLabel>Label</FormFieldLabel>
+          <InputNext defaultValue="Value" data-testid="test-id-1" />
+          <FormFieldHelperText>Helper text</FormFieldHelperText>
+        </FormFieldNext>
+      );
+
+      cy.findByRole("textbox").focus();
```

Comment:
may be better to use `findByLabelText` to find input, which is what's recommended by testing library, as well as testing label association